frontend/src/app/components/resizer/wp-resizer.component.ts

```diff
@@ -136,10 +149,32 @@ export class WpResizerDirective extends UntilDestroyedMixin implements OnInit, A
   }
 
   resizeMove(deltas:ResizeDelta) {
+    // Avoid text-selection while the user is dragging the resizer
+    deltas.origin.preventDefault();
+
     // Get new value depending on the delta
-    // The resizingElement is not allowed to be smaller than 530px
     this.elementWidth = this.elementWidth - deltas.relative.x;
-    let newValue = this.elementWidth < 530 ? 530 : this.elementWidth;
+    let newValue;
+
+    // The resizingElement is not allowed to be smaller than the elementMinWidth
+    if (this.elementWidth < this.elementMinWidth) {
+      newValue = this.elementMinWidth;
+
+      // Show the resizer red when it reaches its limit (min-width)
+      if (this.resizer.style.color !== 'red') {
```

Comment:
Please do not use inline styles in .ts/.js code. Instead check for a class and add/remove this class accordingly. 
I don't think that we already have an existing class, so you may need to add a new one `-error` or whatever you want to call it and use one of our existing style variables for the actual color.

Reply:
Thank you for your feedback @HDinger 

1 - It is flickering because a set a timeout of 500ms to remove the style, so if you drag longer the style is removed and then it is set again. I realize that there is a 'resized' event, so I'll move there the css cleaning to solve this.

2 - Ok, I'll do that. A few questions regarding this:
- Should I place this new '-error' class in the component? I'm trying to @import from _global.scss but I got "SassError: no mixin named exports". How do you use sass variables on your components? I couldn't find any case.
- Do you have any place where you store classes for global use? Should I place this new '-error' class there?

Thanks and excuse so many questions.

Reply:
You have basically two options for the class: either you add it limited the wp resizer component, or you add a globally available class. In my eyes, a global error styling class makes sense as we do not have one yet. In this case, you can add it to the file `_generic.sass`. There, we have all kind of classes that are not limited to a component and useful on multiple places. 

Regarding the variables: The (global) CSS variables that we have, are defined in the `design.rb`. You can use these variables without any import. The syntax slightly differs between angular encapsulated styles and the styles which are lying under `app/assets/stylesheets/`. For the first one, you write `color: var(--variable-name)`. For the later, you use the normal CSS variable syntax: `color: $variable-name`.

So in this case, the following code should be added to the `_generic.sass`. Afterwards you only have to toggle/check for the class name.

```
.-error
  color: $content-form-error-color
```

Don't worry about asking many questions :) Please, let me know if something is still unclear.